designer/apps/api/v1/serializers.py

```diff
@@ -68,28 +118,38 @@ def to_representation(self, value):
 
         return ret
 
+    def get_program_homepage(self, obj):
+        """
+        Get serialized version of program_homepage for the given obj
+        Args:
+            obj: (ProgramPage) The program page currently being serialized.
 
-class ProgramPageSerializer(serializers.ModelSerializer):
-    """
-    Serializer for the Program Page
-    """
-    program_documents = ProgramDocumentField(read_only=True, many=True)
-    branding = BrandingField(read_only=True, many=True)
-    hostname = serializers.SerializerMethodField()
+        Returns:
+            (dict) Serialized version of the program_homepage for the given obj
+        """
+        program_homepage = obj.program_homepage.first()
 
-    class Meta:
-        model = ProgramPage
-        fields = (
-            'id',
-            'uuid',
-            'title',
-            'slug',
-            'last_published_at',
-            'program_documents',
-            'branding',
-            'hostname',
-            'idp_slug',
-        )
+        if not program_homepage:
+            return {}
+
+        ret = {
+            'display': program_homepage.display,
+            'header': program_homepage.header,
+            'description': program_homepage.description,
+            'link': {
+                'display_text': program_homepage.link_display_text,
+                'url': program_homepage.link_url,
+            }
+        }
+        return ret
```

Comment:
Similar nit as above, you could remove the step of creating `ret` and just return the object.